changelog.asc

```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS(4.5) 0.6.4
+** Updated backend MathJax and jQuery libraries
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
+** Improved experience for devices with larger screens
```

Comment:
I'd add "such as chromebook and tablet". Some people may learn that we can be used on chromebook too reading this word, maybe